index.html

```diff
@@ -3725,6 +3725,17 @@ <h2><span class="chunk">cICP</span> Coding-independent code points for video sig
               PNG format, it should be noted that this format is not part of the [[ITU-R-BT.709]] standard.
           </aside>
 
+          <p>If <code>Video Full Range Flag</code> is <code>0</code>
+            (a <a>narrow-range image</a>), recommended practice 
+            is to define transfer functions 
+            such as EOTF or inverse OETF 
```

Comment:
The EOTF acronym doesn't seem to be defined anywhere else in the document. It would be nice to avoid introducing new terminology just to explain how to handle the Full Range flag, but at very least the terminology should be defined

Reply:
Better?